addons/l10n_in_website_sale/models/res_partner.py

```diff
@@ -0,0 +1,23 @@
+from odoo import models
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    def _l10n_in_update_user_sez_status(self, partner_gst_treatment):
```

Comment:
```suggestion
    def _l10n_in_update_user_gst_treatment(self, partner_gst_treatment):
```
now it doesn't just relate to SEZ

Reply:
Still not applied @sham-odoo